libcodechecker/libhandlers/parse.py

```diff
@@ -24,6 +24,7 @@
 from libcodechecker.logger import LoggerFactory
 
 from libcodechecker.output_formatters import twodim_to_str
+from plist_to_html import PlistToHtml
```

Comment:
From libCodeChecker's point of view, `plist_to_html` is an external module. Thus it must come between the system-wide imports and the `libcodechecker` imports.